### PR TITLE
Create a new taglib to capture the save/apply bottom bar

### PR DESCRIPTION
**Pull request by Vlatombe**
This introduces `<f:saveApplyBar/>`, capturing a recurring pattern in our jelly views (originally introduced in fbb2bca313af176ac11ed2e72fd2b838b141832f)

Added save/apply pattern to 
* Computer
* MasterComputer
* LogRecorder

* [x] Needs https://github.com/jenkinsci/acceptance-test-harness/pull/1754

See [JENKINS-XXXXX](https://issues.jenkins.io/browse/JENKINS-XXXXX).

### Testing done
* Checked with various roles (Admin/Manage/ExtendedRead) that configuration screens were displayed the same as before.
* Checked that Save/Apply works as expected for Log Recorder configuration submission.
* Checked that Save/Apply works as expected for Computer configuration submission.
* Checked that Save/Apply works as expected for MasterComputer configuration submission.

### Proposed changelog entries

- Developer: a new taglib `<f:saveApplyBar/>` is now available for configuration forms.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [X] The Jira issue, if it exists, is well-described.
- [X] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly
Comment (Vlatombe) [inline core/src/main/resources/jenkins/model/Jenkins/configure.jelly:63]: Needed because of https://github.com/jenkinsci/jenkins/pull/9813/files#diff-04f3f5f17eaeb24ac6c8f50acae68a658010e18d2de16105942d34e151542fbfR57

File: core/src/main/resources/hudson/model/View/configure.jelly
Comment (Vlatombe) [inline core/src/main/resources/hudson/model/View/configure.jelly:59]: This effectively changes the label `OK` to `Save`. But I don't think there was any good reason for this label to differ from other screens.

File: core/src/main/resources/jenkins/model/Jenkins/MasterComputer/configure.jelly
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/MasterComputer/configure.jelly:58]: Also funny:

> Screenshot 2024-10-11 at 15 12 07

**Review by daniel-beck — DISMISSED** [review]

File: core/src/main/resources/hudson/model/Computer/configure.jelly
Comment (Vlatombe) [reply core/src/main/resources/hudson/model/Computer/configure.jelly:49]: Good catch, fixed by https://github.com/jenkinsci/jenkins/pull/9813/files#diff-581669f62053a1ed8fed271068c074eec26588530194da1aef0d14f722f48b30R1516

File: core/src/main/resources/jenkins/model/Jenkins/MasterComputer/configure.jelly
Comment (Vlatombe) [reply core/src/main/resources/jenkins/model/Jenkins/MasterComputer/configure.jelly:58]: Fixed by https://github.com/jenkinsci/jenkins/pull/9813/files#diff-40583d1d185a48f8484bb0b00ce88d369357e85d1667546a09eb955db42ac846R4162

**Review by NotMyFault — APPROVED** [review]

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
